core/domain/classifier_services_test.py

```diff
@@ -205,7 +205,11 @@ def _mock_get_label_name(l):
             self.string_classifier._get_prediction_report_for_doc(-1))
         self.assertEquals(prediction_report['prediction_label_id'], 1)
 
-    def test_training(self):
+    def test_predict_label_for_doc(self):
+        # Until I come up with larger test data sets, this will do
```

Comment:
You probably shouldn't refer to yourself in a public codebase. This comment will confuse other developers. I recommend rewording it to be more clear on what your thoughts are here. In particular, you're saying that setting the default min docs/labels to predict to 0 will do until you have larger tests (I inferred this; it's better if the comment specifically calls it out). Also, why is selecting '0' adequate? What will larger test data sets do to improve this test or the test of prediction overall? Perhaps this should also be a TODO with what you plan on doing to address whatever the concern is here (the action item being to provide larger test data sets).